src/Main.java

```diff
@@ -1,5 +1,49 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //Задача№1
+        int year = 2023;
+        if (leapYearChecker(year)) {
+            System.out.println(year + " год является високосным.");
+        } else {
+            System.out.println(year + " год не является високосным.");
+        }
+        //Задача№2
+        int system = 1;
+        checkPhone(system, year);
+        //Задача№3
+        int deliveryDistance = 95;
+        delivery(deliveryDistance);
     }
+    public static boolean leapYearChecker (int year) {
```

Comment:
```suggestion
    public static boolean isLeapYear(int year) {
```
методы, которые возвращают boolean, надо начинать с is